india_compliance/gst_india/client_scripts/e_waybill_actions.js

```diff
@@ -498,6 +481,48 @@ function get_generate_e_waybill_dialog(opts, frm) {
     return d;
 }
 
+function get_sub_suppy_type_options(frm) {
+    let options =[]
+    const same_gstin = frm.doc.billing_address_gstin == frm.doc.company_gstin;
+
+    if (frm.doctype === "Delivery Note") {
+
+        if (frm.doc.is_return) {
+            if (same_gstin) {
+                options = ["For Own Use", "Exhibition or Fairs"];
+            } else {
+                options = ["Job Work Returns", "SKD/CKD"];
+            }
+        } else {
+            if (same_gstin) {
+                options = [
+                    "For Own Use",
+                    "Exhibition or Fairs",
+                    "Line Sales",
+                    "Recipient Not Known",
+                ];
+            } else {
+                options = ["Job Work", "SKD/CKD"];
+            }
+        }
+    }
+    if (frm.doctype === "Stock Entry") {
+        options = ["Job Work"];
+    }
+
+    if (frm.doctype === "Subcontracting Receipt") {
+
+        if (frm.doc.is_return) {
+            options = ["Job Work"];
+            }
+        else {
+                options = ["Job Work Returns"];
+            }
+        }
```

Comment:
`Job Work Returns` should come when `is_return` is true, right?

Reply:
Subcontracting receipt will be job work return.
is_return will be again sending it to job worker

Reply:
alright thanks, understood